shared/js/url_helper.js

```diff
@@ -4,13 +4,8 @@ var rscheme = /^(?:[a-z\u00a1-\uffff0-9-+]+)(?::(?:\/\/)?)/i;
 
 var UrlHelper = {
 
-  // Placeholder anchor tag to format URLs.
-  a: null,
-
   getUrlFromInput: function urlHelper_getUrlFromInput(input) {
-    this.a = this.a || document.createElement('a');
-    this.a.href = input;
-    return this.a.href;
+    return new URL(input).href;
```

Comment:
Hi Fabrice,
Sorry I am new to gaia. Can you mentor me?

Do you mean I should test input with:

if (input.indexOf("http://") == 0 || input.indexOf("https://") == 0) {
            return new URL(input).href;                            // Absolute URL
        } else if (input.indexOf(".") != -1) {
            return new URL(document.URL, input).href;  // Relative URL
        } 

Is this correct?